docs/01-Welcome-to-FOME.md

```diff
@@ -1,32 +1,25 @@
----
-sidebar_position: 1
-slug: /
----
-
 # Welcome to FOME
 
 Fome is a new project intending to deliver a user focused high feature ECU with open source hardware options and open firmware that give control back to the user, owner and driver.
 
 As there are many complicated topics to cover we recommend that you start with the topics in the "intro" section to the left.
-The first page to view should probably be [the FOME Overview page](Intro-Start-Here/Overview).
+The first page to view should probably be [the FOME Overview page](01-Intro-Start-Here/Overview).
```

Comment:
Looks like it doesn't like prefixes after all

```suggestion
The first page to view should probably be [the FOME Overview page](Intro-Start-Here/Overview).
```

Reply:
Thanks that fixed the issue